doc/datatable.md

```diff
@@ -0,0 +1,66 @@
+# ASAB WebUI DataTable component
```

Comment:
@titkov49 I miss the information, how one can add a datatable component to an existing project. Also there is no mention here about that it needs to be added to navigation (routes).

It would be great, if you will also mention, that the developer needs to implement axios and define api call(s) to get the data. In demo, you have solved it with mocked ones, but this will not be the case when used in the products.

Also some example of api call with the use of pagination would be great. Now you are paginating through the mocked data, which are static, but this will not be the case in 99% of the usage of this data table component.